contextbasedrestrictionsv1/context_based_restrictions_v1_examples_test.go

```diff
@@ -45,8 +44,8 @@ import (
 // in a configuration file and then:
 // export IBM_CREDENTIALS_FILE=<name of configuration file>
 //
+var _ = Describe(`ContextBasedRestrictionsV1 Examples Tests`, func() {
 
-var _ = Describe(`contextbasedrestrictionsv1.ContextBasedRestrictionsV1 Examples Tests`, func() {
```

Comment:
The package prefix can stay.

Reply:
The package prefixes are removed by the new code generator.